lib/rspec_profiling/collectors/statsd.rb

```diff
@@ -76,18 +77,27 @@ def statsd
           sd.namespace = "ldxe.#{NAMESPACE}.app"
         end
       end
+      
+      def formatDesc(description)
+        ::Digest::SHA1.hexdigest(description)[8..15]
+      end
+
+      def formatFile(path) 
+        path.gsub('/', '.').gsub(/(\.rb$|^\.)/, '')
```

Comment:
No need in parentheses for the regex in last gsub, the can be simply deleted. Or you can use non capturing group, because there is no need to operate with it